freqtrade/wallets.py

```diff
@@ -99,11 +99,21 @@ def _update_dry(self) -> None:
         used_stake = 0.0
 
         if self._config.get("trading_mode", "spot") != TradingMode.FUTURES:
-            current_stake = self.start_cap + tot_profit - tot_in_trades
-            total_stake = current_stake
             for trade in open_trades:
                 curr = self._exchange.get_pair_base_currency(trade.pair)
-                _wallets[curr] = Wallet(curr, trade.amount, 0, trade.amount)
+                used_stake += sum(
+                    o.stake_amount for o in trade.open_orders if o.ft_order_side == trade.entry_side
```

Comment:
Maybe there could be a naming confusion between order stake_amount (quantity / leverage) and amount (quantity)?

Reply:
not sure what you mean? 
technically, it's cost what we're looking for - but cost isn't reliable in futures markets, as different exchanges return different things (with / without leverage)

Reply:
My bad I had to double check that part but it is good for me, thank for the clarification.